runner/copy.go

```diff
@@ -225,7 +226,7 @@ func (m *SyncWatcher) String() string {
 	if !done {
 		return "Job still running."
 	}
-	return "Job completed succesfully."
+	return "Job completed successfully."
```

Comment:
Is this what gets returned to ETCD?
```
if err != nil {
		return fmt.Sprintf("Job failed with error: %v", err)
	}
```

If so, is it possible to make it an error instead of a string?